### DISCUSSION TITLE
Is it possible to disable "security.*" xattrs?

### DISCUSSION THREAD

**Discussion by AM5800** [libfuse/libfuse#818] [Q&A]
I see a lot of "getxattr / security.capability" in my FUSE mount. In fact, there are millions of them. 
Our filesystem does not support security xattrs, so we always respond with ENODATA. 

However, it has a significant impact on the performance of our system. 
Is there a way to disable this feature? We do need other xattrs.

**Comment by trapexit** [comment]
Return -ENOSYS. The kernel will stop sending any xattr calls. Unfortunately, there is no limit to just security.capability calls.

**Comment by bsbernd** [comment]
Yeah it is not beautiful and an all or nothing (all xattr enabled, or none). Problem is that kernel fuse also only receives the string would then filter it on the kernel side. Better would be if kernel fuse could receive the xattr type as enum.
